### DISCUSSION TITLE
Show in the map all the station locations contacted today

### DISCUSSION THREAD

**Discussion by faocrates** [foldynl/QLog#428] [Ideas]
Looking for the ability to show in the map all the station locations contacted today (or at least e.g. the 50 more recent contacts made). I think it's quite satisfying to see in the world map all QSOs managed.

**Comment by foldynl** [comment]
Currently, you can show the contacted stations via Logbook -> Statistics -> Show on Map. You can select the Date Range what you want. I am not sure if this should be displayed on the online map as a new layer. But maybe it wouldn't be bad. I will think about it. If so, I would have to redesign the Online Map menu a bit, because it is getting too long.

**Reply by faocrates** [reply]
Thank you for considering it.

**Reply by bchurchwell** [reply]
This is exactly what I was looking for to be able to make maps of past POTA activations, thank you.
Two suggestions:
Add an option for a time range within the day for days with more than one activation. Filtering by 6 digit grid square works for this but some entities are so close together that the 6 digit grid won't change.
Add an option to show lines for the QSOs.
